config/vufind/Primo.ini

```diff
@@ -99,7 +129,6 @@ dateRange[] = creationdate
 [Facets]
 tlevel  = "Limit To"
 rtype   = "Format"
-creator = "Author"
```

Comment:
Is author faceting no longer possible, or did it never work to begin with?

Reply:
At least I can't make it work with the old or the new API, so I believe it's better to leave it out. It also seems that the Central Discovery Index contains some pretty wildly differently written author names, which might contribute to it not being usable. I suspect author facet would only work when searching Primo's local index, but since CDI is the normal use case, I'd leave it out.

Reply:
I added it back but commented out and with a comment.